roles/tas-install/charts/intel-telemetry-aware-scheduler/templates/tas-health-policy.yml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.taspolicy.create }}
```

Comment:
Do we want a configurable policy created during creation? It doesn't seem useful given this is a dummy policy. Better to leave this out I think.